triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -163,14 +163,21 @@ object TriggerServiceFixture {
       a <- testFn(uri, client, ledgerProxy)
     } yield a
 
-    fa.onComplete { _ =>
```

Comment:
Ah, so the problem was that the shutdown here would race with the second use of `withTriggerService` in the test?

Reply:
Yes.